src/js/components/Form/Form.js

```diff
@@ -569,9 +570,10 @@ const Form = forwardRef(
         };
       };
 
-      return { useFormField, useFormInput };
+      return { useFormField, useFormInput, kindArg };
```

Comment:
I think the context should call it "kind". I think "*Arg" should primarily be used within the function component to distinguish from other variables in scope. Any reason we cannot just call it "kind" everywhere?

Reply:
agree Ill change it back to `kind`